### ISSUE TITLE
Contributing

### ISSUE THREAD

**Issue by dmateusp** [munnellg/OfMagesAndMagic#1]
I would like to contribute to the project, do you have any TODO list or any ideas that you want to implement ?

Can you add a CONTRIBUTING.md ?

**Comment by dmateusp** [comment]
Some parts should use/be inspired from #5 `README.md`

**Comment by munnellg** [comment]
Again, I'll organise this later in the week. I will be meeting with the coordinator for the club that I wrote the game in conjunction with on Wednesday. Hopefully we'll identify the strengths and weaknesses of what we have so far which will give us a decent set of necessary contributions along with relative priorities

**Comment by coolharsh55** [comment]
I think the most (arguably) important ToDo for this project is documentation - a nice guide explaining what the project is, what parts are for kids, and what parts are the teachers (or demonstrators) responsibility. And then how it is structured, where to tinker to get something changed, etc. 

Apart from that, a wiki page maintained to list ideas without turning them into issues, @munnellg ?

**Comment by munnellg** [comment]
Definitely! Documentation is a massive weakness here. There are a number of items which need to be written up:

1. Usage document - getting started, rules of the game, etc.
2. Student coding guide - how to write a bot of your own
3. Configuration guide - methods by which a teacher can tweak the game to suit their teaching style
4. Teaching guide - recommendations for how the game could be used in a teaching environment
5. Code documentation - for enthusiastic developers who want to delve into the core of the program

If we are maintaining a Wiki, then it should be the Wiki attached to this github repository. Best to keep everything in one place if possible.

**Comment by dmateusp** [comment]
+1 on everything you said, it is very hard to contribute at the moment but it is normal on a project that was individual until now.

This should be the main focus as it will open doors for many more contributors that have interests in the project.
